contribs/gmf/src/services/themesservice.js

```diff
@@ -103,24 +120,51 @@ gmf.Themes.findTheme_ = function(themes, themeName) {
  * @return {angular.$q.Promise} Promise.
  */
 gmf.Themes.prototype.getBgLayers = function() {
+
   goog.asserts.assert(!goog.isNull(this.promise_));
   return this.promise_.then(goog.bind(
       /**
        * @param {gmf.ThemesResponse} data The "themes" web service response.
-       * @return {Array.<Object>} Array of background layer objects.
+       * @return {angular.$q.Promise}
        */
       function(data) {
-        var bgLayers = data['background_layers'].map(goog.bind(function(item) {
+        var configs = data['background_layers'];
+        this.backgroundLayerConfigs_ = configs;
+        var promises = configs.map(goog.bind(function(item) {
+          return this.$http_.get(item.url, {
+            cache: false
```

Comment:
I suppose we want this kind of request to always receive a "fresh" response.  After being logged out, if a user refreshes the page, we want this request to always be made and not use the cache that would return "the user is connected" when he's not.

Does that make sense ?